app/containers/message/Image.js

```diff
@@ -55,6 +59,21 @@ export default class extends Component {
 		this.setState({ isPressed: state });
 	}
 
+	setLoading = (loading) => {
+		this.setState({ loading });
+	}
+
+	renderLoading() {
+		const { loading } = this.state;
+		if (loading) {
+			return (
+				<View style={styles.loading}>
```

Comment:
I suppose wrapping ActivityIndicator in View is not necessary here

Reply:
It is to centre it.

Reply:
yeah, but you can remove View and add styles.loading to ActivityIndicator directly and see that nothing has changed :)